### PR TITLE
[issue #443] make the keep screen on in the setting

### PR DESCRIPTION
**Pull request by jakeoneijk**
Hello I add the "keep screen on" in the setting.
If there's anything I should do more, please let me know
tnx:)

### CODE REVIEW

File: app/src/main/res/values/string-not_translatable.xml [hunk 1]
```diff
@@ -41,6 +41,7 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="linkbox_default_filename" translatable="false">linkbox.txt</string>
     <string name="web_browser" translatable="false">Web browser</string>
     <string name="app_drawer_launcher_special_files_description" translatable="false">App drawer launchers: LinkBox, ToDo, QuickNote</string>
+    <string name="keep_screen_on_description" translatable="false">Keep the screen on</string>
```
Comment (gsantner) [inline app/src/main/res/values/string-not_translatable.xml]: You can remove that, in this case title/desc is the same so no need

**Comment by jakeoneijk** [comment]
I fix the code as you said :)

**Comment by gsantner** [comment]
@jakeoneijk Thanks, merged!
Can you please delete your fork and create a fresh one? Else you carry on old commits for next PR (if you plan to do so). Thats the easiest way, but if you know howto deal with feature branches and/or rebase thats also fine :D

**Comment by gsantner** [comment]
@jakeoneijk @jdckr @noob893796  @jakubAerik @MikeMikerson 
Thanks for your recent code contribution to Markor!
Next release is thought to be release this weekend. Do you have some uncommited / non-merge-request changes left that you want to get in or fix before? Just let me know :smile: .

**As always, happy about any help, doing open source projects alone is not that fun. Development community is the heart of the project ;).**
If you want to spend some time on Markor, there's lots of things open (issue tracker) and you probably also have some ideas what to add/improve. Always open to new stuff ;).

My personal next goal in the project is: To remove the whole autoname stuff from Markor. Just one big pile of legacy code from Writeily that even after much time put in not really works. Started here with the new `new file dialog` which allows to put in filename, which takes the filename logics out of editor.

BTW, if you are not yet on the CONTRIBUTORS.md list, you can always send the details via a PR. I just reused an existing PR to let you know, guess thats not a problem right? :D.

**Pull request merged**